src/app.css

```
@@ -0,0 +1,278 @@
+@import 'tailwindcss';
+
+/* Dark mode: use html.palette-dark class (matches tokens.css and config.tsx) */
+@variant dark (&:where(html.palette-dark, html.palette-dark *));
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@variant" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)